assets/memes.css

```
@@ -0,0 +1,255 @@
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'SF Pro Display', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
+    background-color: #667eea; /* Fallback for gradient */  
+    background: linear-gradient(135deg, #667eea 0%, #764ba2 100%);
```

Biome (1.9.4)

[error] Unexpected shorthand property background after background-color



(lint/suspicious/noShorthandPropertyOverrides)